lib/the_little_thinkers_space/file_size_checker.ex

```diff
@@ -16,4 +16,11 @@ defmodule TheLittleThinkersSpace.FileSizeChecker do
         {:error, :cannot_stat_file}
     end
   end
+
+  def get_file_size(path) do
+    case File.stat(path) do
+      {:ok, %{size: size}} -> size
+      _ -> {:error, :cannot_stat_file}
```

Comment:
If you can return an `:error` tuple then the success type should be an `:ok` tuple